lib/Common/Memory/RecyclerWriteBarrierManager.h

```diff
@@ -124,6 +145,32 @@ class RecyclerWriteBarrierManager
 public:
     static void WriteBarrier(void * address);
     static void WriteBarrier(void * address, size_t bytes);
+    static void ToggleBarrier(void * address, size_t bytes, bool enable);
+    static BOOL IsBarrierAddress(void * address);
+    static BOOL IsBarrierAddress(uintptr_t index);
+    static void VerifyIsBarrierAddress(void * address, size_t bytes);
+    static void VerifyIsNotBarrierAddress(void * address, size_t bytes);
+    static void VerifyIsBarrierAddress(void * address);
+    static bool Initialize();
```

Comment:
the linker should drop them, but yes, should make them only applicable for test build

---
In reply to: [86644302](https://github.com/Microsoft/ChakraCore/pull/1920#discussion_r86644302) [](ancestors = 86644302)